Source/Core/DolphinWX/NetPlay/NetPlayLauncher.cpp

```diff
@@ -92,32 +92,22 @@ bool NetPlayLauncher::Join(const NetPlayJoinConfig& config)
   }
 }
 
-const std::string NetPlayLaunchConfig::DEFAULT_TRAVERSAL_HOST = "stun.dolphin-emu.org";
-
-std::string
-NetPlayLaunchConfig::GetTraversalHostFromIniConfig(const IniFile::Section& netplay_section)
+std::string NetPlayLaunchConfig::GetTraversalHost()
 {
-  std::string host;
-
-  netplay_section.Get("TraversalServer", &host, DEFAULT_TRAVERSAL_HOST);
-  host = StripSpaces(host);
+  std::string host = StripSpaces(Config::Get(Config::NETPLAY_TRAVERSAL_SERVER));
 
   if (host.empty())
-    return DEFAULT_TRAVERSAL_HOST;
+    return Config::NETPLAY_DEFAULT_TRAVERSAL_HOST;
 
   return host;
 }
 
-u16 NetPlayLaunchConfig::GetTraversalPortFromIniConfig(const IniFile::Section& netplay_section)
+u16 NetPlayLaunchConfig::GetTraversalPort()
 {
-  std::string port_str;
-  unsigned long port;
-
-  netplay_section.Get("TraversalPort", &port_str, std::to_string(DEFAULT_TRAVERSAL_PORT));
-  StrToWxStr(port_str).ToULong(&port);
+  u16 port = Config::Get(Config::NETPLAY_TRAVERSAL_PORT);
```

Comment:
Same question: could NETPLAY_DEFAULT_TRAVERSAL_PORT be set as the default value for this config info?